webpack/assets/javascripts/react_app/components/LoginPage/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
I think that even though it's not a connected component, we should still keep our current folder structure convention where the `ComponentName.js` is the component file and the `index.js` is the integration file.

In your case, there is no integration rather than exporting the main component so I see it as:
1. `LoginPage/LoginPage.js` - The component
2. `LoginPage/index.js` - `export default from './LoginPage';`

Reply:
added the beloved folder structure :+1: